### PR TITLE
Removes JSHeroes from past editions

### PR DESCRIPTION
**Pull request by SaraVieira**

### CODE REVIEW

File: app/data/menuitems.js [hunk 1]
```diff
@@ -32,12 +32,12 @@ export default [
       {
         id: 6.1,
         url: 'http://2017.jsheroes.io/',
-        label: 'JSHeroes 2017',
+        label: '2017',
       },
       {
         id: 6.2,
         url: '/2018',
```
Comment (alexnm) [inline app/data/menuitems.js]: can you use next/link for relative urls to avoid full page reload?

Comment (SaraVieira) [reply app/data/menuitems.js]: I had a key here that says internal?

Comment (SaraVieira) [reply app/data/menuitems.js]: Done :)

File: app/components/nav.js [hunk 2]
```diff
@@ -57,7 +57,13 @@ export default class Nav extends Component {
                   <ul className="menu-items inside">
                     {item.links.map(link => (
                       <li key={link.id}>
-                        <a href={`${link.url}`}>{link.label}</a>
+                        {link.url.startsWith('http') ? (
```
Comment (SaraVieira) [inline app/components/nav.js]: See here

**Pull request merged**